packages/desktop/src/shared/types.ts

```diff
@@ -5,5 +5,11 @@ export interface ProjectData {
   engine: EngineData
   app: {
     sketchesDir: string
+    // TODO: activeSketchId should be part of the save state
+    // but causing errors when Hedron opens directly on a sketch
+    // activeSketchId: string | null
+    selectedNodes: { [sketchId: string]: string }
+    selectedInputs: { [inputId: string]: string }
+    openedParamGroups: { [sketchId: string]: Record<number, boolean> }
```

Comment:
These bits of state are now saved to the project file, so when you load your project back up, it has you where you left off in terms of opened/closed stuff